apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -20,13 +20,34 @@ const alphabetMapper =  [
   commonMsg.answerOptionG(),
 ];
 
+const calculateNotAnswered = (multipleChoiceDataArr) => {
+    let total = 0;
+  multipleChoiceDataArr.forEach (studentsAnswersObj => {
+        if (studentsAnswersObj.percentAnswered) {
+            total += studentsAnswersObj.percentAnswered;
+        }
+    });
+
+    return (100 - total);
+};
+
 const answerColumnsFormatter = (percentAnswered, {rowData, columnIndex, rowIndex, property}) => {
   const cell = rowData.answers[columnIndex - 1];
+
+  let percentValue = 0;
+
+  if (property === 'notAnswered') {
+     percentValue = calculateNotAnswered(rowData.answers);
+  } else {
+     percentValue = (cell && cell.percentAnswered) || '-';
```

Comment:
If you set `percentValue` to "-" rather than to a number here, `MultipleChoiceAnswerCell` will be confused because it's expecting that propType to be a number. You have a few choices: 
1.) Instead of "-" you could set `percentValue` to 0, then have `MultipleChoiceAnswerCell` just render "0%" 
2.) Instead of "-" you could set `percentValue` to 0. Then, you could introduce some logic in `MultipleChoiceAnswerCell`  if percentValue === 0 render "-"
3.) Adjust the propTypes in `MultipleChoiceAnswerCell` to use oneOfType, explained [here](https://reactjs.org/docs/typechecking-with-proptypes.html)

Which option are you leaning toward? If you want to chat through the pros/cons of each let me know.